lib/response-handlers/foods-response-handler.js

```diff
@@ -0,0 +1,19 @@
+const url = `https://protected-basin-11627.herokuapp.com/`
+
+const handleResponse = require('../response-handlers/initial-response-handler')
+
+const postNewFood = (name, calories) => {
+  let newFood = { food: {'name': name, 'calories': calories}}
+
+  let postOptions = {
+    method: "POST",
+    body: JSON.stringify(newFood),
+    headers: {'Content-Type': 'application/json'},
+  }
+  fetch(`${url}/api/v1/foods`, postOptions)
```

Comment:
I don't know if I agree with this fetch in the response handlers file? I think the code is clean, but I'm not sure I understand the refactor.  I can ask you about it later :)

Reply:
Ah, actually, I meant to abstract this into foods-fetch-requests. I'll move it in a soon-coming PR 😃